src/tanstar/config.rs

```diff
@@ -35,12 +38,14 @@ impl TConfig {
         max_process_time: Duration,
         turning_radius: f32,
         v_merge_threshold: f32,
+        virtualize_flyzone: bool,
     ) -> Self {
         Self {
             buffer_size: buffer_size,
             max_process_time: max_process_time,
             turning_radius: turning_radius,
             vertex_merge_threshold: v_merge_threshold,
+            virtualize_flyzone: virtualize_flyzone,
```

Comment:
Do you use clippy?

I get lots of _suggestions_ to use the Field Init shorthand (i.e. `buffer_size` instead of `buffer_size: buffer_size`).